app/src/main/java/com/example/habitsmasher/PublicPrivateButtons.java

```diff
@@ -76,52 +66,51 @@ public PublicPrivateButtons(View view, boolean isPublic){
      * only one button may be clicked at a time.
      */
     public void setClickListeners(){
-        // on click listeners and logic behind the buttons switching
-        // TODO: Check interactions when the selected button is selected again
-        // preferably make it unselectable
-
+        // track whether the button is selected or not
         _publicButton.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View view) {
                 // if we check the public box, uncheck the private box
-                if (_privateSelected){
-                    _privateButton.performClick();
-                    _privateSelected = false;
+                if (_publicSelected){
+                    _publicSelected = false;
+                }
+                else {
                     _publicSelected = true;
```

Comment:
Changed the logic to just track whether the buttons selected. Now both buttons or no buttons can be selected.